packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -121,11 +156,18 @@ const { densityClass } = useDensity(props);
   @include layers.component() {
     @include define-borders();
 
+    --onyx-table-column-hover: #26628d30;
```

Comment:
```suggestion
    --onyx-table-column-hover: color-mix(
      in srgb,
      var(--onyx-color-base-neutral-300),
      transparent 50%
    );
```

We can use color-miz to transparentize CSS variables so we can use existing colors and also don't have to define special colors for dark mode :)

The values above are just examples, they don't look perfect

Reply:
I'll talk with Jannick about the topic :)